inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -817,8 +834,56 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         if (!quantized) {
             return;
         }
+        if (LayerInfo(layer).isWeightable() || LayerInfo(layer).isEltwise()) {
+            printed_properties.emplace_back(
+                "weights scale factor", std::to_string(quantized->_weights_quant.GetScale()));
+            if (quantized->_weights_quant.IsStatsSet()) {
+                for (auto& min : quantized->_weights_quant.GetMinValues()) {
+                    printed_properties.emplace_back(
+                        "weights min val", std::to_string(min));
+                }
+                for (auto& max : quantized->_weights_quant.GetMaxValues()) {
+                    printed_properties.emplace_back(
+                        "weights max val", std::to_string(max));
+                }
+            }
+
+            if (quantized->_bias_quant.IsStatsSet()) {
+                for (auto& min : quantized->_bias_quant.GetMinValues()) {
+                    printed_properties.emplace_back(
+                        "bias min val", std::to_string(min));
+                }
+                for (auto& max : quantized->_bias_quant.GetMaxValues()) {
+                    printed_properties.emplace_back(
+                        "bias max val", std::to_string(max));
+                }
+            }
+        }
         printed_properties.emplace_back(
-            "scale factor", std::to_string(quantized->_dst_quant.GetScale()));
+            "src scale factor", std::to_string(quantized->_src_quant.GetScale()));
+        if (quantized->_src_quant.IsStatsSet()) {
+            for (auto &min : quantized->_src_quant.GetMinValues()) {
+                printed_properties.emplace_back(
+                    "src min val", std::to_string(min));
+            }
+            for (auto& max : quantized->_src_quant.GetMaxValues()) {
+                printed_properties.emplace_back(
+                    "src max val", std::to_string(max));
+            }
+        }
+
+        printed_properties.emplace_back(
+            "dst scale factor", std::to_string(quantized->_dst_quant.GetScale()));
+        if (quantized->_dst_quant.IsStatsSet()) {
+            for (auto& min : quantized->_dst_quant.GetMinValues()) {
+                printed_properties.emplace_back(
+                    "dst min val", std::to_string(min));
+            }
+            for (auto& max : quantized->_dst_quant.GetMaxValues()) {
+                printed_properties.emplace_back(
+                    "dst max val", std::to_string(max));
+            }
+        }
     });
```

Comment:
Maybe it's reasonable to make this lambda function not-lambda, since it's quite long now and LoadNetwork() method itself is too long and hard readable.

Reply:
good suggestion